src/app/util/ClientMonitoringRegistrationTable.h

```diff
@@ -19,33 +19,33 @@
 
 #include <app-common/zap-generated/cluster-objects.h>
 #include <lib/core/CHIPConfig.h>
+#include <lib/core/CHIPPersistentStorageDelegate.h>
 #include <lib/support/CodeUtils.h>
-#include <platform/CHIPDeviceConfig.h>
 
 namespace chip {
 class ClientMonitoringRegistrationTable
 {
 public:
     using MonitoringRegistrationStruct = chip::app::Clusters::ClientMonitoring::Structs::MonitoringRegistration::Type;
 
-    ClientMonitoringRegistrationTable(FabricIndex fabricIndex);
-    ~ClientMonitoringRegistrationTable(){};
+    struct ClientRegistrationEntry : MonitoringRegistrationStruct
+    {
+        bool isValid() { return clientNodeId != 0 && ICid != 0 && fabricIndex != 0; }
```

Comment:
```suggestion
        bool IsValid() { return clientNodeId != 0 && ICid != 0 && fabricIndex != 0; }
```

Also, should some of those 0 be various `kInvalid...` or `kUndefined...` values?

Reply:
done